src/test/java/org/folio/marc4ld/authority/field110/MarcToLdAuthorityEmptyField110IT.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.marc4ld.authority.field110;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+
+import org.folio.marc4ld.Marc2LdTestBase;
+import org.junit.jupiter.api.Test;
+
+public class MarcToLdAuthorityEmptyField110IT extends Marc2LdTestBase {
```

Comment:
Couldn't it be a part of MarcToLdAuthorityJurisdictionOrganization110IT class?

Reply:
This is not specific to MARC 110. Is this test really needed?

Reply:
looks like it's no longer needed